### ISSUE TITLE
Unable to install a module from PSGallery

### ISSUE THREAD

**Issue by some-sh** [PowerShell/PowerShell#16541]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://github.com/PowerShell/PowerShell/releases)
- [X] Search the existing issues.
- [X] Refer to the [FAQ](https://github.com/PowerShell/PowerShell/blob/master/docs/FAQ.md).
- [X] Refer to [Differences between Windows PowerShell 5.1 and PowerShell](https://docs.microsoft.com/powershell/scripting/whats-new/differences-from-windows-powershell).

### Steps to reproduce

Tried installing the module [SNMP](https://www.powershellgallery.com/packages/SNMP/1.0.0.1) from [PowerShellGallery](https://www.powershellgallery.com/). The below error was thrown:
``` PowerShell
➜ ~  Install-Module -Name SNMP
Install-Module: The 'Install-Module' command was found in the module 'PowerShellGet', but the module could not be loaded. For more information, run 'Import-Module PowerShellGet'.
```
So I tried 'Import-Module PowerShellGet`, the below error was thrown:
``` PowerShell
➜ ~  Import-Module PowerShellGet
Import-Module: Errors occurred while loading the format data file:
C:\program files\powershell\7\Modules\PackageManagement\PackageManagement.format.ps1xml, , C:\program files\powershell\7\Modules\PackageManagement\PackageManagement.format.ps1xml: The file was skipped because of the following validation exception: File C:\program files\powershell\7\Modules\PackageManagement\PackageManagement.format.ps1xml cannot be loaded because its operation is blocked by software restriction policies, such as those created by using Group Policy..
```
Tried several fixes like setting execution policy, changed the 'SecurityProtocol' according to this [announcement](https://devblogs.microsoft.com/powershell/powershell-gallery-tls-support/), nothing worked

I've been facing this issue for quite some time now. Please let me know if anymore detail is required. 
``` PowerShell
Name                           Value
----                           -----
PSVersion                      7.2.0
PSEdition                      Core
GitCommitId                    7.2.0
OS                             Microsoft Windows 10.0.19044
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

### Expected behavior

```console
PS> Install-Module -Name SNMP
... Module installed successfully (Not sure about the actual output)
```

### Actual behavior

```console
PS> Install-Module -Name SNMP
Install-Module: The 'Install-Module' command was found in the module 'PowerShellGet', but the module could not be loaded. For more information, run 'Import-Module PowerShellGet'
```

### Error details

```console
PSMessageDetails      :
Exception             : System.Management.Automation.CommandNotFoundException: The 'Install-Module' command was found in the module 'PowerShellGet', but
                        the module could not be loaded. For more information, run 'Import-Module PowerShellGet'.
                         ---> System.Management.Automation.CmdletInvocationException: Errors occurred while loading the format data file:
                        C:\program files\powershell\7\Modules\PackageManagement\PackageManagement.format.ps1xml, , C:\program
                        files\powershell\7\Modules\PackageManagement\PackageManagement.format.ps1xml: The file was skipped because of the following
                        validation exception: File C:\program files\powershell\7\Modules\PackageManagement\PackageManagement.format.ps1xml cannot be
                        loaded because its operation is blocked by software restriction policies, such as those created by using Group Policy..

                         ---> System.Management.Automation.RuntimeException: Errors occurred while loading the format data file:
                        C:\program files\powershell\7\Modules\PackageManagement\PackageManagement.format.ps1xml, , C:\program
                        files\powershell\7\Modules\PackageManagement\PackageManagement.format.ps1xml: The file was skipped because of the following
                        validation exception: File C:\program files\powershell\7\Modules\PackageManagement\PackageManagement.format.ps1xml cannot be
                        loaded because its operation is blocked by software restriction policies, such as those created by using Group Policy..

                           at System.Management.Automation.Runspaces.InitialSessionState.ThrowTypeOrFormatErrors(String resourceString, String errorMsg,
                        String errorId)
                           at System.Management.Automation.Runspaces.InitialSessionState.UpdateFormats(ExecutionContext context, Boolean update)
                           at System.Management.Automation.Runspaces.InitialSessionState.Bind_UpdateFormats(ExecutionContext context, Boolean updateOnly)
                           at System.Management.Automation.Runspaces.InitialSessionState.<>c__DisplayClass130_0.<Bind>b__1()
                           at System.Management.Automation.Runspaces.InitialSessionState.Bind(ExecutionContext context, Boolean updateOnly, PSModuleInfo
                        module, Boolean noClobber, Boolean local, Boolean setLocation)
                           at Microsoft.PowerShell.Commands.ModuleCmdletBase.LoadModuleManifest(String moduleManifestPath, ExternalScriptInfo
                        manifestScriptInfo, Hashtable data, Hashtable localizedData, ManifestProcessingFlags manifestProcessingFlags, Version
                        minimumVersion, Version maximumVersion, Version requiredVersion, Nullable`1 requiredModuleGuid, ImportModuleOptions& options,
                        Boolean& containedErrors)
                           --- End of inner exception stack trace ---
                           at System.Management.Automation.Runspaces.PipelineBase.Invoke(IEnumerable input)
                           at System.Management.Automation.PowerShell.Worker.ConstructPipelineAndDoWork(Runspace rs, Boolean performSyncInvoke)
                           at System.Management.Automation.PowerShell.CoreInvokeHelper[TInput,TOutput](PSDataCollection`1 input, PSDataCollection`1
                        output, PSInvocationSettings settings)
                           at System.Management.Automation.PowerShell.CoreInvoke[TInput,TOutput](PSDataCollection`1 input, PSDataCollection`1 output,
                        PSInvocationSettings settings)
                           at System.Management.Automation.PowerShell.Invoke[T](IEnumerable input, IList`1 output, PSInvocationSettings settings)
                           at System.Management.Automation.PowerShell.Invoke[T]()
                           at System.Management.Automation.CommandDiscovery.AutoloadSpecifiedModule(String moduleName, ExecutionContext context,
                        SessionStateEntryVisibility visibility, Exception& exception)
                           --- End of inner exception stack trace ---
                           at System.Management.Automation.CommandDiscovery.TryModuleAutoDiscovery(String commandName, ExecutionContext context, String
                        originalCommandName, CommandOrigin commandOrigin, SearchResolutionOptions searchResolutionOptions, CommandTypes commandTypes,
                        Exception& lastError)
                           at System.Management.Automation.CommandDiscovery.LookupCommandInfo(String commandName, CommandTypes commandTypes,
                        SearchResolutionOptions searchResolutionOptions, CommandOrigin commandOrigin, ExecutionContext context)
                           at System.Management.Automation.ExecutionContext.CreateCommand(String command, Boolean dotSource)
                           at System.Management.Automation.PipelineOps.AddCommand(PipelineProcessor pipe, CommandParameterInternal[] commandElements,
                        CommandBaseAst commandBaseAst, CommandRedirection[] redirections, ExecutionContext context)
                           at System.Management.Automation.PipelineOps.InvokePipeline(Object input, Boolean ignoreInput, CommandParameterInternal[][]
                        pipeElements, CommandBaseAst[] pipeElementAsts, CommandRedirection[][] commandRedirections, FunctionContext funcContext)
                           at System.Management.Automation.Interpreter.ActionCallInstruction`6.Run(InterpretedFrame frame)
                           at System.Management.Automation.Interpreter.EnterTryCatchFinallyInstruction.Run(InterpretedFrame frame)
TargetObject          : Install-Module
CategoryInfo          : ObjectNotFound: (Install-Module:String) [], CommandNotFoundException
FullyQualifiedErrorId : CouldNotAutoloadMatchingModule
ErrorDetails          :
InvocationInfo        : System.Management.Automation.InvocationInfo
ScriptStackTrace      : at <ScriptBlock>, <No file>: line 1
PipelineIterationInfo : {}
```

### Environment data

```powershell
PS> $PSVersionTable
Name                           Value
----                           -----
PSVersion                      7.2.0
PSEdition                      Core
GitCommitId                    7.2.0
OS                             Microsoft Windows 10.0.19044
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

**Comment by iSazonov** [comment]
Please create new issue in https://github.com/PowerShell/PowerShellGet

**Comment by daxian-dbw** [comment]
The exception is thrown from the following code that is checking on whether the system is locked down.
Does your computer have a system-wide lock-down through WLDP or AppLocker?  /cc @TravisEz13 

https://github.com/PowerShell/PowerShell/blob/64d2fe26f3957eeb400e48234accbc8e643ef929/src/System.Management.Automation/security/SecurityManager.cs#L170-L180

**Comment by some-sh** [comment]
> Does your computer have a system-wide lock-down through WLDP or AppLocker

@daxian-dbw  I'm sorry, I don't know how to check this. May I know?

**Comment by TravisEz13** [comment]
This code pass is only when you have AppLocker disallow policies.
You really need to work with your IT department.  
These are the documents to get the Group Policy, which is how AppLocker policy is applied.
https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/gpresult
`gpresult /h <path to report.html>`

**Comment by TravisEz13** [comment]
Here is the documentation about AppLocker script policy:
https://docs.microsoft.com/en-us/windows/security/threat-protection/windows-defender-application-control/applocker/script-rules-in-applocker

**Comment by TravisEz13** [comment]
Adding needs triage to make sure the security WG reviews this.

**Comment by PaulHigin** [comment]
@WG-Security
We agree the above investigation.  It is not clear what policy is being applied so it is not possible to comment more, except that the policy requires that PowerShell denies loading PowerShellGet.  We recommend that you investigate the AppLocker policy applied to the system.
